website/static/_redirects

```diff
@@ -1,3 +1,11 @@
+
+# Redirects for quickstarts sections that were split
+
+/quickstarts/manual-install?step=2 /quickstarts/manual-install?step=2 301
```

Comment:
@mirnawong1 I'm going to remove the redirects so it won't affect future links to the correct steps. Also, not sure we can do redirects at this level
```suggestion
```

Reply:
awesome! should we do a crawler trigger after this pr is merged @runleonarun ?

Reply:
@mirnawong1 probably a good idea. I can try out our new process!